README.md

```diff
@@ -167,6 +163,30 @@ py.test --numprocesses=4 --looponfail --maxfail=1
 pytest -n 4 -f --maxfail=1
 ```
 
+#### How to Execute the Tests?
+
+1. Make sure that you have `leveldb` and `tox` installed
+
+```sh
+brew install leveldb
+pip install tox
```

Comment:
This is redundant if you've already run https://github.com/ethereum/web3.py/#developer-setup (and crucially leaves other things not installed, if you skipped that step).

Maybe just link back to the dev setup section, instead?